vllm/model_executor/model_loader.py

```diff
@@ -23,6 +23,11 @@ def _set_default_torch_dtype(dtype: torch.dtype):
 
 def _get_model_architecture(config: PretrainedConfig) -> Type[nn.Module]:
     architectures = getattr(config, "architectures", [])
+    # Special handling for quantized Mixtral.
```

Comment:
wait i don't see a clause searching for `if quantization is not None` here?

Reply:
Good catch. Let me fix it right away.